### PR TITLE
VRT and Visual Testing UI changes

### PR DESCRIPTION
**Pull request by prashelke**
and Also handled Create baseline selected then check point action should get passed with info MSG Baseline uploaded, Please approve it on VRT dashboard to create baseline

Thank you for your contribution.
Before submitting this PR, please make sure:

- [x] PR description and commit message should describe the changes done in this PR
- [x] Verify the PR is pointing to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [x] Latest Code from master or specific release branch is merged to your branch
- [x] No unwanted\commented\junk code is included
- [x] No new warning upon build solution
- [x] Code Summary\Comments are added to my code which explains what my code is doing
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for New and Existing Functionality
- [x] Verify that changes are compatible with all relevant browsers and platforms.
- [x] After creating pull request there should not be any conflicts
- [ ] Resolve all Codacy comments
- [x] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments
- [ ] Update the Help Library document to match any feature changes

### CODE REVIEW

File: Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs [hunk 1]
```diff
@@ -320,7 +319,15 @@ private void TrackVRT()
                     switch (result.Status)
                     {
                         case TestRunStatus.New:
-                            mAct.Error += $"No baseline found, Please approve it on dashboard to create baseline.{System.Environment.NewLine}{result.Url}";
+                            if(mAct.CreateBaselineImage)
+                            {
+                                mAct.ExInfo += $"Baseline uploaded, Please approve it on VRT dashboard to create baseline.{System.Environment.NewLine}{result.Url}";
```
Comment (Maheshkale447) [inline Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs:324]: Remove to create baseline from message

File: Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs [hunk 2]
```diff
@@ -320,7 +319,15 @@ private void TrackVRT()
                     switch (result.Status)
                     {
                         case TestRunStatus.New:
-                            mAct.Error += $"No baseline found, Please approve it on dashboard to create baseline.{System.Environment.NewLine}{result.Url}";
+                            if(mAct.CreateBaselineImage)
```
Comment (Maheshkale447) [inline Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs:322]: Also make sure action is not failing on baseline upload

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs (hunk 1 above)
Comment (prashelke) [reply Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs:324]: Done

File: Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs (hunk 2 above)
Comment (prashelke) [reply Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs:322]: Tested

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**